src/main/java/transforms/DateTransform.java

```diff
@@ -42,18 +43,10 @@
     @JsonCreator
     public DateTransform(
             @JsonProperty( Constants.PATTERN ) String[] pattern,
-            @JsonProperty( Constants.TIMEZONE ) String timezone
+            @JsonProperty( Constants.TIMEZONE ) Optional<String> timezone
     ) {
         this.pattern = pattern;
-        this.timezone = TimeZone.getTimeZone( timezone );
-    }
-
-    @JsonCreator
-    public DateTransform(
-            @JsonProperty( Constants.PATTERN ) String[] pattern
-    ) {
-        this.pattern = pattern;
-        this.timezone = TimeZone.getTimeZone( "America/New_York" );
+        this.timezone = TimeZone.getTimeZone( timezone.orElse("America/New_York") );
```

Comment:
@jokedurnez this shouldn't be extracted to string constant but approving for now.